deployments/apiv2/manifests/apiv2-dep.yaml

```diff
@@ -0,0 +1,57 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: apiv2
+  labels:
+    app: apiv2
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: apiv2
+  template:
+    metadata:
+      labels:
+        app: apiv2
+        version: latest
+    spec:
+      containers:
+      # - name: users
+      #   image: 077346344852.dkr.ecr.us-east-2.amazonaws.com/users
+      #   imagePullPolicy: Always
+      #   ports:
+      #   - containerPort: 8000
+      - name: mantarray
+        image: 077346344852.dkr.ecr.us-east-2.amazonaws.com/mantarray
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 9000
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: apiv2-ingress
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.org/rewrites: serviceName=mantarray rewrite=/
+status:
+  loadBalancer: {}
+spec:
+  rules:
+  - host: apiv2.curibio-test.com
+    http:
+      paths:
+      # - path: /users
+      #   pathType: Prefix
+      #   backend:
+      #     service:
+      #       name: users
+      #       port:
+      #         number: 80
+      - path: /mantarray/
```

Comment:
I don't' think you need the trailing '/'

Reply:
I was running across an issue where if I didn't use the trailing `/` then it would rewrite `/mantarray/firmware_latest` to `//firmware_latest`. Not sure if there is a better way to solve that issue, but this seemed to work